manim/mobject/mobject.py

```diff
@@ -2333,12 +2333,39 @@ def get_family(self, recurse: bool = True) -> list[Self]:
             >>> gr.get_family()
             [Group, VGroup(Square, Rectangle), Square, Rectangle, Mobject, VMobject]
 
+        See also
+        --------
+        :meth:`~.Mobject.family_members_with_points`, :meth:`~.Mobject.align_data`
+
         """
         sub_families = [x.get_family() for x in self.submobjects]
         all_mobjects = [self] + list(it.chain(*sub_families))
         return remove_list_redundancies(all_mobjects)
 
     def family_members_with_points(self) -> list[Self]:
+        """Filters the list of family members (generated by :meth:`.get_family`) to include only mobjects with points.
+
+        Returns
+        -------
+        list
```

Comment:
Could it be, instead,
```suggestion
        list[Mobject]
```
?